### DISCUSSION TITLE
Defined size is not maintained SOLVED

### DISCUSSION THREAD

**Discussion by Ulisssses** [arkypita/LaserGRBL#1998] [Q&A]
Hello, I have a question that cannot be resolved. When flashing the GRBL firmware, many things have been misconfigured for me. Some I have been able to solve, but not this one.
If I introduce in the program a JPG size of 20x18.7 mm for example, and I get a size of approximately 62x58 mm.

I can't find the possible solution.

**Comment by Tribo-2dot0** [comment]
configure seus passos/mm em $100 e $101... o valor mais comum é 80

**Reply by Ulisssses** [reply]
It is set at 250,000 steps/min if I put 80 it doesn't even move...

My complete configuration is:
$0=10
$1=25
$2=0
$3=1
$4=0
$5=0
$6=0
$10=1
$11=0.010
$12=0.002
$13=0
$20=0
$21=1
$22=0
$23=0
$24=25.000
$25=500.000
$26=250
$27=1.000
$30=1000
$31=0
$32=0
$100=250.000
$101=250.000
$102=250.000
$110=3000.000
$111=3000.000
$112=3000.000
$120=10.000
$121=10.000
$122=10.000
$130=200.000
$131=200.000
$132=200.000

**Comment by Ulisssses** [comment]
the manufacturer has given me the correct data thank you
